src/Opdex.Platform.WebApi/Controllers/VaultGovernancesController.cs

```diff
@@ -303,7 +368,7 @@ public async Task<ActionResult<TransactionQuoteResponseModel>> QuoteCompleteProp
     [ProducesResponseType(typeof(TransactionQuoteResponseModel), StatusCodes.Status200OK)]
     [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
-    public async Task<ActionResult<TransactionQuoteResponseModel>> QuoteWithdrawVote([FromRoute] Address address, [FromRoute] ulong proposalId,
+    public async Task<ActionResult<TransactionQuoteResponseModel>> QuoteWithdrawVote([FromRoute] Address address, ulong proposalId,
```

Comment:
Consider keeping [FromRoute] on ProposalId

Reply:
Wasn't intention to remove it, adding it back